app/script/calling/CallMessageMapper.js

```diff
@@ -22,55 +22,45 @@
 window.z = window.z || {};
 window.z.calling = z.calling || {};
 
-z.calling.CallMessageMapper = (function() {
+z.calling.CallMessageMapper = {
   /**
    * Map incoming call message into entity.
    *
    * @private
    * @param {Object} event - Call event object
-   * @returns {CallMessage} Call message entity
+   * @returns {z.calling.entities.CallMessageEntity} Call message entity
    */
-  const _mapEvent = function(event) {
+  mapEvent(event) {
     const {content: callMessage, conversation: conversationId, from: userId, sender: clientId, time} = event;
 
-    const additionalProperties = {
-      clientId: clientId,
-      conversationId: conversationId,
-      time: time,
-      userId: userId,
-    };
+    const additionalProperties = {clientId, conversationId, time, userId};
 
     let content = undefined;
     switch (callMessage.type) {
       case z.calling.enum.CALL_MESSAGE_TYPE.GROUP_SETUP:
       case z.calling.enum.CALL_MESSAGE_TYPE.UPDATE: {
-        const {dest_clientid, dest_userid, props: properties, sdp} = callMessage;
-
-        content = {
-          dest_client_id: dest_clientid,
-          dest_user_id: dest_userid,
+        const {
+          dest_clientid: destinationClientId,
+          dest_user_id: destinationUserId,
           props: properties,
-          sdp: sdp,
-        };
+          sdp,
+        } = callMessage;
+
+        content = {destinationClientId, destinationUserId, properties, sdp};
```

Comment:
May be something wrong here. I can't understand.

Reply:
No, it is all good. I destructure the format agreed between clients, user proper and explicit camelCase naming in our internal entities and ensure spec compliance in the `toJson` of the `CallMessageEntity`.